beacon-chain/rpc/prysm/v1alpha1/beacon/validators_test.go

```diff
@@ -1675,66 +1673,6 @@ func TestServer_GetValidatorParticipation_OrphanedUntilGenesis(t *testing.T) {
 	assert.DeepEqual(t, wanted, res.Participation, "Incorrect validator participation respond")
 }
 
-func TestServer_GetValidatorParticipation_CurrentAndPrevEpochWithBits(t *testing.T) {
```

Comment:
1. This test looks fine except sync committee part. Is there any reason to remove the whole test?
2. If we decided to remove this test, `runGetValidatorParticipationCurrentAndPrevEpoch` also can be removed.